debug_toolbar/settings.py

```diff
@@ -35,6 +35,11 @@
     ),
     'PROFILER_MAX_DEPTH': 10,
     'SHOW_TEMPLATE_CONTEXT': True,
+    'SKIP_TEMPLATE_PREFIXES': (
+        'debug_toolbar/',
+        'django/forms/widgets/',
+        'admin/widgets',
```

Comment:
Missing trailing slash. If it's intentionally omitted, the docs change is wrong instead :)